package.json

```diff
@@ -35,7 +35,7 @@
     "ci:zip": "zip /tmp/agora-app.zip -qr9 * .[^.]*",
     "data:local:mongo": "npm run data:local && npm run mongo:import && npm run mongo:create:indexes",
     "mongo:import": "sh ./scripts/mongo-import.sh",
-    "mongo:create:indexes": "mongo --host localhost ./scripts/mongo-create-Indexes.js",
+    "mongo:create:indexes": "mongosh --host localhost ./scripts/mongo-create-Indexes.js",
```

Comment:
For MongoDB 6.0+, `mongo` was replaced with `mongosh`, see [here](https://stackoverflow.com/a/74240679)